examples/manifests/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: observatorium-api
+  name: observatorium-api
+  namespace: observatorium
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: observatorium-api
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: observatorium-api
+    spec:
+      containers:
+      - args:
+        - --web.listen=0.0.0.0:8080
+        - --metrics.ui.endpoint=http://127.0.0.1:9091/
+        - --metrics.query.endpoint=http://127.0.0.1:9091/api/v1/query
```

Comment:
Interesting. If you allow these to be configured to they have independent listeners? 
I guess it's not really a Kubernetes manifest question though...

Reply:
They have a single listener but difference reverse proxies. Router dispatches requests accordingly.

https://github.com/observatorium/observatorium/blob/24b2fcd3824b9aed464a0978d942c4ec4fbb7abe/internal/server/server.go#L60

A proxy:

https://github.com/observatorium/observatorium/blob/24b2fcd3824b9aed464a0978d942c4ec4fbb7abe/internal/proxy/proxy.go#L33-L68

We'll see how it's work in real life :D